src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const dataOfState = [];
```

Comment:
Try to make up a more descriptive name. For example `stateHistory`

Reply:
Why you ignored this without any comment?

Reply:
I'm sorry. I just didn't notice it.